src/bundle/Controller/SearchController.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUiBundle\Controller;
+
+use eZ\Publish\Core\Repository\SearchService;
+use EzSystems\EzPlatformAdminUi\Form\Data\Search\SimpleSearchData;
+use EzSystems\EzPlatformAdminUi\Form\Type\Search\SimpleSearchType;
+use EzSystems\EzPlatformAdminUi\Tab\Dashboard\PagerContentToDataMapper;
+use Pagerfanta\View\TwitterBootstrap4View;
+use Symfony\Component\HttpFoundation\Request;
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion;
+use eZ\Publish\API\Repository\Values\Content\Query\SortClause;
+use eZ\Publish\API\Repository\Values\Content\Query;
+use Pagerfanta\Pagerfanta;
+use eZ\Publish\Core\Pagination\Pagerfanta\ContentSearchAdapter;
+use Symfony\Component\HttpFoundation\Response;
+
+class SearchController extends Controller
+{
+    /** @var SearchService */
+    private $searchService;
+
+    /** @var PagerContentToDataMapper */
+    protected $pagerContentToDataMapper;
+
+    /**
+     * @param SearchService $searchService
+     * @param PagerContentToDataMapper $pagerContentToDataMapper
+     */
+    public function __construct(
+        SearchService $searchService,
+        PagerContentToDataMapper $pagerContentToDataMapper
+    ) {
+        $this->searchService = $searchService;
+        $this->pagerContentToDataMapper = $pagerContentToDataMapper;
+    }
+
+    /**
+     * Renders the simple search form and search results.
+     *
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function searchAction(Request $request): Response
+    {
+        $limit = $request->query->getInt('limit', 10);
+        $page = $request->query->getInt('page', 1);
+        $query = $request->query->get('query');
+
+        $form = $this->createForm(SimpleSearchType::class, new SimpleSearchData($limit, $page, $query));
+
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            /** @var SimpleSearchData $data */
+            $data = $form->getData();
+
+            $query = new Query();
+            $query->filter = new Criterion\LogicalAnd(
+                [
+                    new Criterion\Visibility(Criterion\Visibility::VISIBLE),
+                    new Criterion\FullText($data->getQuery()),
+                ]
+            );
+            $query->sortClauses[] = new SortClause\DateModified(Query::SORT_ASC);
+
+            $pagerfanta = new Pagerfanta(
+                new ContentSearchAdapter(
+                    $query,
+                    $this->searchService
+                )
+            );
+
+            $pagerfanta->setMaxPerPage($limit);
+            $pagerfanta->setCurrentPage($page);
+
+            $urlGenerator = $this->get('router');
```

Comment:
Please inject in the constructor. We tend to not use container directly in the controllers.